bears/python/PyUnusedCodeBear.py

```diff
@@ -15,7 +15,7 @@ class PyUnusedCodeBear(LocalBear):
     CAN_DETECT = {'Unused Code'}
 
     def run(self, filename, file,
-            remove_all_unused_imports: bool=False,
+            remove_all_unused_imports: bool=True,
             remove_unused_variables: bool=True):
         """
         Detects unused code. By default this functionality is limited to:
```

Comment:
@mridubhatnagar No! Don't forget it! It's there in the `.coafile` 😄 I only remembered it from that `coala` PR... But here it's also set... I had not enough sleep last night! 😉